lib/03_simon_says.rb

```diff
@@ -0,0 +1,37 @@
+def echo(word)
+  word
```

Comment:
Hmm, I think that you want a `puts ` here but perhaps the exercise only asked for a return value?